submissions/mod-comparison/mod-comparison.md

```diff
@@ -0,0 +1,106 @@
+
+![Making plastic](plastic.png)
+
+It has a very different feel to vanilla, and whilst sometimes it feels like things are being made complex for the sake of complexity, and often feels overwhelming, I did enjoy the challenge. The enhanced builds are very nice too – the belts show an excellent example of this. In this modpack, there are six tiers of belts, instead of the 3 from vanilla, each requiring the one from the tier before, plus an additional piece which gets more advanced with each level. For example, a tier three belt requires a tier two belt, plus bronze and steel, a tier six belt requires a tier five belt plus 3 types of nitinol, which only comes from very advanced metal processing, but runs 2.5 times faster than the tier 3. 
+
+![All belt tiers](allbelts.png)
+
+You also get the same sort of tiers with trains, inserters, assembling machines, bots, exoskeletons and so on – compensating for the added complexity and throughput required with massively overpowered facilities to help you deal with it all. 
+
+I think this mod probably has the most basic art of the three with different tiers of buildings generally just being distinguished by a palette swap, however it definitely works and looks more than good enough, it never bothered me while I was playing.
+
+![A smelting area](angelbobs.png)
+
+All in all, I found the modpack generally well balanced. There were definitely some difficulty spikes, especially when I first tried to make plastic, and I struggled with biters until I discovered how good the plasma cannon is, but generally I enjoyed my playthrough, and found it decently challenging. As is always the case, I definitely have some ideas for what I would do differently if I played again!
+
+### Industrial Revolution 2
+
+The next mod on my list is [Industrial Revolution 2](https://mods.factorio.com/mod/IndustrialRevolution). The intent behind this mod is that you work through a series of different tech levels as you progress – starting off with burner and steam power initially, then moving on to normal electrical systems, then unlocking more advanced machines that operate using lasers. We got off to a good start with this mod, having to run steam pipes to all the assembly machines and inserters was an interesting challenge and something that made a nice change to how we were used to thinking about things. Once we moved to the electrical age, deciding when it was worth pulling up the old, steam powered machines to replace them with electrical ones was also interesting, and provided us with an interesting upgrade path that was a bit more involved than the normal “drag an upgrade planner over everything” method from vanilla. However, unfortunately, whilst it was interesting thinking about the upgrades in a few areas, needing to do them across large areas of the factory did get rather repetitive.
```

Comment:
You can mention here that this mod is created by Deadlock989, who is known for creating some of the most beautiful graphics in the Factorio modding scene.